### ISSUE TITLE
Any insights about Pre-training with CoCo or ImageNet 

### ISSUE THREAD

**Issue by YubinXie** [matterport/Mask_RCNN#696]
Does anybody know what is the data bias in coco and ImageNet? What is their unique advantage? 

E.g., which one is better at small objects? which one is better at dense objects? which one is better at faces?

In my data, I feel that coco 1) tends to give a big mask instead of several small ones as imageNet does for a crowded image;  2) COCO gives a smaller training loss value when using the same setting.
Thanks!

**Comment by fastlater** [comment]
@YubinXie  I am thinking to run some test to compare the results using imagenet and using coco. Before that, did you find out something more recently?

**Comment by YubinXie** [comment]
@fastlater You can run both and choose the best one for your data. My previous finding is that Res50 is better than Res101. People mentioned that only Res50 in this Maskrcnn is pre-trained (101 not). Sometimes COCO is better than ImageNet, while sometimes ImageNet is better.

**Comment by fastlater** [comment]
@YubinXie Thank you for your reply. I will do some tests and if I discover something, I will come back here so.

**Comment by JordanMakesMaps** [comment]
@fastlater, any results?

**Comment by kimile599** [comment]
I am running On a small dataset like 200 pics for a plastic residual segment withask rcnn. tried with the initial weights with imagenet, But not Good as coco.
